src/Symfony/Component/Translation/IdentityTranslator.php

```diff
@@ -18,7 +18,14 @@
  */
 class IdentityTranslator implements TranslatorInterface
 {
+    /**
+     * @var MessageSelector
```

Comment:
to be removed

Reply:
Why? I think it's good to typehint variables in docblock?

Reply:
It does not add anything because we already now this with the constructor.